### PR TITLE
Create 2020-04-30-wechaty-chat-room_manager.md

### PR DESCRIPTION
**Pull request by tianyugeng**
发表博客

Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [ ] Add Blog Header, including title, author, date.
e.g.
```
---
 
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```
- [ ] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [ ] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [ ] Add `<!--more-->` section to just show the abstract in blog homepage.
- [ ] Add teaser image for the blog
- [ ] Add your self as an author to the author data file at https://github.com/Chatie/blog/blob/master/docs/_data/authors.yml with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/Chatie/blog#contribute-guideline)

### CODE REVIEW

File: _posts/2020-04-30-wechaty-chat-room-manager.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+title: "微信群管辅助"
+author: tianyugeng
+date: '2020年4月30日'
+header:
+  teaser: /assets/images/bio-photo.png
```
Comment (lijiarui) [inline _posts/2020-04-30-wechaty-chat-room-manager.md]: Thanks for this blog!

Please teaser image for the blog, I found you add teaser here but cannot find the image, please add an image at `/assests/2020/room-manager/bio-photo.png`

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _posts/2020-04-30-wechaty-chat-room-manager.md (hunk 1 above)
Comment (tianyugeng) [reply _posts/2020-04-30-wechaty-chat-room-manager.md]: can i delete the teaser image if i do not want it?

Comment (tianyugeng) [reply _posts/2020-04-30-wechaty-chat-room-manager.md]: i mean ,let my blog has no teaser image.

Comment (lijiarui) [reply _posts/2020-04-30-wechaty-chat-room-manager.md]: I think you can find an image for this blog, then the blog may seem more beautiful.

File: _posts/2020-04-30-wechaty-chat-room-manager.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+---
+title: "微信群管辅助"
+author: tianyugeng
+date: '2020年4月30日'
```
Comment (lijiarui) [inline _posts/2020-04-30-wechaty-chat-room-manager.md]: should change to: `2020-4-30 09:00 +0800`

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change the date as I request, thanks!

**Comment by lijiarui** [comment]
Congratulations! :tada: :tada: :tada:

You become a wechaty contributor successfully!  :clap: :clap: :clap:

Please: 
- [ ] add my wechat: `ruirui_0914` and I will invite you into Wechaty Contributor Room
- [ ] join wechaty org by clicking https://github.com/wechaty

image

**Pull request merged**